<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form3</class>
 <widget class="QWidget" name="Form3">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>540</width>
    <height>330</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>540</width>
    <height>330</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>540</width>
    <height>330</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Ustawienia</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff/>
   </iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="margin">
    <number>4</number>
   </property>
   <item row="1" column="0">
    <widget class="QPushButton" name="ApplyB">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>22</height>
      </size>
     </property>
     <property name="text">
      <string>Zastosuj</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>1</string>
      </attribute>
      <widget class="QSlider" name="sR">
       <property name="geometry">
        <rect>
         <x>490</x>
         <y>10</y>
         <width>25</width>
         <height>91</height>
        </rect>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="focusPolicy">
        <enum>Qt::WheelFocus</enum>
       </property>
       <property name="maximum">
        <number>100</number>
       </property>
       <property name="value">
        <number>100</number>
       </property>
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
      <widget class="QCheckBox" name="bB">
       <property name="geometry">
        <rect>
         <x>360</x>
         <y>60</y>
         <width>91</width>
         <height>51</height>
        </rect>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="text">
        <string>Przesuwaj
razem</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="playSB">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>70</y>
         <width>141</width>
         <height>22</height>
        </rect>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="text">
        <string>Odtwarzaj dźwięk</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="playSExB">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>90</y>
         <width>321</width>
         <height>22</height>
        </rect>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="text">
        <string>Odtwarzaj dźwięk podczas eksportu do pliku</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="visEffB">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>50</y>
         <width>331</width>
         <height>22</height>
        </rect>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="text">
        <string>Używaj efektów dźwiękowych do wizualizacji</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="titleEB">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>230</y>
         <width>461</width>
         <height>22</height>
        </rect>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="text">
        <string>Posyłaj do QMPExport tytuł utworu zamiast nazwy pliku piosenki</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="chChnB">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>130</y>
         <width>161</width>
         <height>25</height>
        </rect>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="text">
        <string>Zmiksuj do mono</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="savEffB">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>371</width>
         <height>22</height>
        </rect>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="text">
        <string>Zapisuj efekty dźwiękowe podczas eksportu do pliku</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="zChnB">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>190</y>
         <width>211</width>
         <height>22</height>
        </rect>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="text">
        <string>Zamień lewy i prawy kanał</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>330</x>
         <y>164</y>
         <width>131</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Priorytet wyższy</string>
       </property>
      </widget>
      <widget class="QSlider" name="pSL">
       <property name="geometry">
        <rect>
         <x>290</x>
         <y>122</y>
         <width>31</width>
         <height>102</height>
        </rect>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="toolTip">
        <string>Priorytet wątku odtwarzającego i nagrywającego</string>
       </property>
       <property name="maximum">
        <number>2</number>
       </property>
       <property name="value">
        <number>1</number>
       </property>
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="tickPosition">
        <enum>QSlider::TicksBelow</enum>
       </property>
      </widget>
      <widget class="QRadioButton" name="chChn2B">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>150</y>
         <width>161</width>
         <height>25</height>
        </rect>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="text">
        <string>Zmiksuj do stereo</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>330</x>
         <y>119</y>
         <width>191</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Priorytet czasu rzeczywistego</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="chChnNotB">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>110</y>
         <width>111</width>
         <height>25</height>
        </rect>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="text">
        <string>Nie miksuj</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>330</x>
         <y>210</y>
         <width>131</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Priorytet wysoki</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="volB">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>291</width>
         <height>22</height>
        </rect>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="text">
        <string>Używaj głośności z QMPout w programie</string>
       </property>
      </widget>
      <widget class="QSlider" name="sL">
       <property name="geometry">
        <rect>
         <x>460</x>
         <y>10</y>
         <width>25</width>
         <height>91</height>
        </rect>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="focusPolicy">
        <enum>Qt::WheelFocus</enum>
       </property>
       <property name="maximum">
        <number>100</number>
       </property>
       <property name="value">
        <number>100</number>
       </property>
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
      <widget class="QCheckBox" name="bEffB">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>170</y>
         <width>241</width>
         <height>21</height>
        </rect>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="text">
        <string>Ustaw głośność przed efektami</string>
       </property>
      </widget>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>380</x>
         <y>20</y>
         <width>61</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text">
        <string>Głośność
QMPout:</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="silenceB">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>210</y>
         <width>251</width>
         <height>22</height>
        </rect>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="text">
        <string>Wyciszaj podczas zatrzymywania</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>2</string>
      </attribute>
      <widget class="QCheckBox" name="userBitsB">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>130</y>
         <width>221</width>
         <height>22</height>
        </rect>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="text">
        <string>Wymuś ilość bitów na próbkę:</string>
       </property>
      </widget>
      <widget class="QGroupBox" name="bitsB">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>160</y>
         <width>131</width>
         <height>71</height>
        </rect>
       </property>
       <property name="title">
        <string>Format: Integer</string>
       </property>
       <widget class="QRadioButton" name="bits8B">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>20</y>
          <width>51</width>
          <height>23</height>
         </rect>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="text">
         <string>8</string>
        </property>
       </widget>
       <widget class="QRadioButton" name="bits16B">
        <property name="geometry">
         <rect>
          <x>70</x>
          <y>20</y>
          <width>51</width>
          <height>23</height>
         </rect>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="text">
         <string>16</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QRadioButton" name="bits24B">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>40</y>
          <width>51</width>
          <height>23</height>
         </rect>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="text">
         <string>24</string>
        </property>
       </widget>
       <widget class="QRadioButton" name="bits32B">
        <property name="geometry">
         <rect>
          <x>70</x>
          <y>40</y>
          <width>51</width>
          <height>23</height>
         </rect>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="text">
         <string>32</string>
        </property>
       </widget>
      </widget>
      <widget class="QCheckBox" name="userRateB">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>261</width>
         <height>22</height>
        </rect>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="text">
        <string>Wymuś częstotliwość próbkowania:</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="userChnB">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>70</y>
         <width>171</width>
         <height>22</height>
        </rect>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="text">
        <string>Wymuś ilość kanałów:</string>
       </property>
      </widget>
      <widget class="QSpinBox" name="rateB">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>40</y>
         <width>81</width>
         <height>23</height>
        </rect>
       </property>
       <property name="minimum">
        <number>4000</number>
       </property>
       <property name="maximum">
        <number>384000</number>
       </property>
       <property name="value">
        <number>44100</number>
       </property>
      </widget>
      <widget class="QSpinBox" name="chnB">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>100</y>
         <width>81</width>
         <height>23</height>
        </rect>
       </property>
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>2</number>
       </property>
       <property name="value">
        <number>2</number>
       </property>
      </widget>
      <widget class="QLabel" name="label_5">
       <property name="geometry">
        <rect>
         <x>270</x>
         <y>10</y>
         <width>171</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Rozmiar bufora dźwięku:</string>
       </property>
      </widget>
      <widget class="QLabel" name="bsL">
       <property name="geometry">
        <rect>
         <x>450</x>
         <y>10</y>
         <width>61</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>000000</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="buffQB">
       <property name="geometry">
        <rect>
         <x>270</x>
         <y>70</y>
         <width>201</width>
         <height>22</height>
        </rect>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="text">
        <string>Włącz dodatkowy bufor</string>
       </property>
      </widget>
      <widget class="QLabel" name="mbsL">
       <property name="geometry">
        <rect>
         <x>450</x>
         <y>30</y>
         <width>61</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>000000</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_8">
       <property name="geometry">
        <rect>
         <x>270</x>
         <y>30</y>
         <width>171</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Rozmiar  bufora w QMP:</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="titleHB">
       <property name="geometry">
        <rect>
         <x>200</x>
         <y>190</y>
         <width>311</width>
         <height>22</height>
        </rect>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="text">
        <string>Pokazuj tylko kiedy główne okno jest ukryte</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="titleB">
       <property name="geometry">
        <rect>
         <x>190</x>
         <y>170</y>
         <width>311</width>
         <height>22</height>
        </rect>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="text">
        <string>Pokazuj powiadomienie o zmianie tytułu</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_6">
       <property name="geometry">
        <rect>
         <x>270</x>
         <y>145</y>
         <width>171</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Powieksz bufor dźwięku</string>
       </property>
      </widget>
      <widget class="QSpinBox" name="soundBB">
       <property name="geometry">
        <rect>
         <x>440</x>
         <y>141</y>
         <width>41</width>
         <height>23</height>
        </rect>
       </property>
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>8</number>
       </property>
       <property name="value">
        <number>4</number>
       </property>
      </widget>
      <widget class="QSpinBox" name="QBufB">
       <property name="geometry">
        <rect>
         <x>300</x>
         <y>110</y>
         <width>41</width>
         <height>23</height>
        </rect>
       </property>
       <property name="minimum">
        <number>2</number>
       </property>
       <property name="maximum">
        <number>64</number>
       </property>
       <property name="value">
        <number>4</number>
       </property>
      </widget>
      <widget class="QLabel" name="label_7">
       <property name="geometry">
        <rect>
         <x>485</x>
         <y>145</y>
         <width>16</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>x</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_9">
       <property name="geometry">
        <rect>
         <x>350</x>
         <y>115</y>
         <width>121</width>
         <height>18</height>
        </rect>
       </property>
       <property name="text">
        <string>x bufor dźwięku</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="buffQIfNeadedB">
       <property name="geometry">
        <rect>
         <x>300</x>
         <y>90</y>
         <width>211</width>
         <height>22</height>
        </rect>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="text">
        <string>Użyj tylko kiedy wymagany</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_10">
       <property name="geometry">
        <rect>
         <x>270</x>
         <y>50</y>
         <width>171</width>
         <height>18</height>
        </rect>
       </property>
       <property name="text">
        <string>Dodatkowy bufor QMP:</string>
       </property>
      </widget>
      <widget class="QLabel" name="bufQL">
       <property name="geometry">
        <rect>
         <x>450</x>
         <y>50</y>
         <width>62</width>
         <height>18</height>
        </rect>
       </property>
       <property name="text">
        <string>nie</string>
       </property>
      </widget>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>tabWidget</tabstop>
  <tabstop>volB</tabstop>
  <tabstop>savEffB</tabstop>
  <tabstop>visEffB</tabstop>
  <tabstop>playSB</tabstop>
  <tabstop>playSExB</tabstop>
  <tabstop>chChnNotB</tabstop>
  <tabstop>chChnB</tabstop>
  <tabstop>chChn2B</tabstop>
  <tabstop>bEffB</tabstop>
  <tabstop>zChnB</tabstop>
  <tabstop>silenceB</tabstop>
  <tabstop>titleEB</tabstop>
  <tabstop>pSL</tabstop>
  <tabstop>bB</tabstop>
  <tabstop>sL</tabstop>
  <tabstop>sR</tabstop>
  <tabstop>ApplyB</tabstop>
  <tabstop>userRateB</tabstop>
  <tabstop>rateB</tabstop>
  <tabstop>userChnB</tabstop>
  <tabstop>chnB</tabstop>
  <tabstop>userBitsB</tabstop>
  <tabstop>bits8B</tabstop>
  <tabstop>bits16B</tabstop>
  <tabstop>bits24B</tabstop>
  <tabstop>bits32B</tabstop>
  <tabstop>buffQB</tabstop>
  <tabstop>buffQIfNeadedB</tabstop>
  <tabstop>QBufB</tabstop>
  <tabstop>soundBB</tabstop>
  <tabstop>titleB</tabstop>
  <tabstop>titleHB</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
